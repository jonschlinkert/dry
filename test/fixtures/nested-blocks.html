{% extends "nested-blocks" %}

{% block "a" %}
before `a.b`: "nested.blocks.html"
  {% block "b" %}
  a.b: "nested.blocks.html"
  {% endblock %}
after `a.b`: "nested.blocks.html"
{% endblock %}

{% block prepend="c" %}
c: "nested.blocks.html"
{% endblock %}

{% block append="c" %}
c: "nested.blocks.html"
{% endblock %}

{% block "c.d" %}
c.d: "nested.blocks.html" outside
{% endblock %}


{% block "e" %}
{% block "f" %}
e.f: "nested.blocks.html" inside
{% endblock %}
{% endblock %}

{% block "e.f" %}
e.f: "nested.blocks.html" outside
{% endblock %}


{% block "g.h" %}
g.h: "nested.blocks.html" outside
{% endblock %}

{% block "g" %}
{% block "h" %}
g.h: "nested.blocks.html" inside
{% endblock %}
{% endblock %}

{% block prepend="i" %}
i: "nested.blocks.html" before
{% block append="j" %}
i.j: "nested.blocks.html" inside
{% endblock %}
i: "nested.blocks.html" after
{% endblock %}
