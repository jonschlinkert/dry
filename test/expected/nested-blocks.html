
before `a.b`: "nested.blocks.html"
a.b: "nested.blocks.html"
after `a.b`: "nested.blocks.html"

a.b: "nested.blocks.html"

b: default

c: default before
c.d: "nested.blocks.html" outside
c: default after
c: "nested.blocks.html"
